src/metabase/public_settings.clj

```diff
@@ -281,6 +281,17 @@
   :type       :string
   :default    "Metabase")
 
+(defsetting loading-message
+  (deferred-tru "Message to show while a query is running.")
+  :visibility :public
+  :type       :keyword)
```

Comment:
Why is this a `:keyword`? Shouldn't this be a `:string`?

Reply:
`Doing science...` or whatever should be a `:string`.

Reply:
it saves the keys from this map
```javascript
export const LOADING_MESSAGE_BY_SETTING = {
  "doing-science": t`Doing science...`,
  "running-query": t`Running query...`,
  "loading-results": t`Loading results...`,
};
```
and the associated values will get recognized and into the pot file.